apps/condo/domains/notification/constants/constants.js

```diff
@@ -651,9 +651,15 @@ const MESSAGE_TYPES = Object.keys(MESSAGE_META)
 
 const MESSAGE_DELIVERY_STRATEGY_AT_LEAST_ONE_TRANSPORT = 'atLeastOneTransport'
 const MESSAGE_DELIVERY_STRATEGY_ALL_TRANSPORTS = 'allTransports'
+
 const MESSAGE_DELIVERY_DEFAULT_PRIORITY = 'default'
 const MESSAGE_DELIVERY_SLOW_PRIORITY = 'slow'
 const MESSAGE_DELIVERY_FAST_PRIORITY = 'fast'
+const MESSAGE_DELIVERY_PRIORITY_TO_TASK_QUEUE_MAP = {
+    [MESSAGE_DELIVERY_FAST_PRIORITY]: 'high',
+    [MESSAGE_DELIVERY_DEFAULT_PRIORITY]: 'medium',
```

Comment:
Actually I just found a bug. I forgot to remap queue name provided from `applyAsync` method with `TASK_QUEUE_REMAPPING`... 
So that's line doesn't work and fail with error in runtime. 
Anyway, you have two options right now:

1. Change default task priority constant to default task name from package (also you can reuse constant value exported from that package)
```suggestion
    [MESSAGE_DELIVERY_DEFAULT_PRIORITY]: DEFAULT_QUEUE_NAME,
```
2. Add remapping logic same as a code inside `registerTaskQueues` method